src/cart.html

```diff
@@ -313,7 +313,11 @@ <h1>YOUR CART</h1>
         </div>
         </div>
             <div id="subtotal" class="carttext"></div>
-            <div><button class="order_now_button">ORDER NOW</button></div>
+            <div>
+              <a href="/krishiconnect/src/payment.html">
```

Comment:
provide relative path over here `./payment.html` it should work fine then. Right now its not visible.

Reply:
Try clicking on the deploy link and see if its working:

Link: https://vercel.live/open-feedback/krishiconnect-git-fork-varsani2520-p-8dae51-anushkabhs-projects.vercel.app?via=pr-comment-visit-preview-link&passThrough=1

Push your changes and then try accessing this link.

Reply:
ok